phoenix_liveview/phoenix_liveview.exs

```diff
@@ -0,0 +1,134 @@
+Application.put_env(:phoenix, :json_library, Jason)
+Application.put_env(:esbuild, :version, "0.12.18")
+
+Mix.install(
+  deps = [
+    {:phoenix, "~> 1.6.0-rc.0", override: true},
+    :jason,
+    :plug_cowboy,
+    {:phoenix_live_view, "~> 0.16.0"},
+    {:esbuild, "~> 0.2"}
+  ]
+)
+
+Code.require_file("mix_install_deps_backport.exs")
+
+Application.put_env(:esbuild, :default,
```

Comment:
This part has been a bit of pain ; one must not move this before `Mix.install/2`, or the defaults are actually erased, and the user gets cryptic error messages due to incorrect (incomplete) `esbuild` CLI call.